lib/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_prepare_conf.pm

```diff
@@ -206,7 +204,7 @@ sub pipeline_analyses {
       -language => 'python3',
       -analysis_capacity => 1,
       -failed_job_tolerance => 100,
-      -rc_name        => 'default',
+      -rc_name        => 'normal',
```

Comment:
is it `normal` or `default`?

Reply:
default is for short quick jobs (<1h), I changed it to "normal" which can be up to 24h (needed in this case for some assemblies)

Reply:
thanks!